### DISCUSSION TITLE
Showing 1st Player

### DISCUSSION THREAD

**Discussion by Lunarisyn104** [PlaceholderAPI/PlaceholderAPI#1020] [(2) Support] [answered]
Hello, I want to use Placeholders and I am using them, Tho I want to Show the Top player of an Placeholder wich is named statistic_fish_caught and I want to show him in My Scoreboard (simplescore is the plugin) and I want to show the 1st Place of The fish caught Placeholder and I didnt find how to do that, It would be very Helpfull if someone Could Help :)

**Comment by Andre601** [comment]
Neither PlaceholderAPI nor the statistics extension provide a leaderboard functionality.

Tho, there is AJLeaderboards, which allows a leaderboard system with pretty much any placeholder from PlaceholderAPI. Maybe give that a look.

**Reply by Lunarisyn104** [reply, marked as answer]
Thankyou For the Suggestion, It works Now! I forgot to Add the %statistic_fish_caught% to ajlb so it didnt recognize it ofc Thankyou so so much!
